src/main/kotlin/dev/cbyrne/kdiscordipc/core/util/temporaryDirectory.kt

```diff
@@ -1,4 +1,4 @@
 package dev.cbyrne.kdiscordipc.core.util
 
 val temporaryDirectory =
-    System.getenv("XDG_RUNTIME_DIR") ?: System.getenv("TMPDIR") ?: System.getenv("TMP") ?: System.getenv("TMPDIR")
```

Comment:
Why was this removed?

Reply:
While looking through the code I saw this and thought it was not necessary to check "TMPDIR" twice. If it isn't a mistake, why is it necessary here?

Reply:
Oh oops, an oversight on my part.